### PR TITLE
Add pluggable EP shared libraries to manylinux python wheel

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 
Add pluggable EP shared libraries to manylinux python wheel

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -201,6 +198,7 @@ def run(self):
 
 if is_manylinux:
     data = ['capi/libonnxruntime_pywrapper.so'] if nightly_build else []
+    data += [path.join('capi', x) for x in dl_libs if path.isfile(path.join('onnxruntime', 'capi', x))]
```
Comment (jywu-msft) [inline setup.py:201]: quick question. i didn't see that capi/libonnxruntime_pywrapper.so in the ort-gpu-nightly package I was testing. is that expected?

**Review by jywu-msft — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (snnn) [reply setup.py:201]: Yes. It is for PyOP. We didn't enable it.

**Comment by snnn** [comment]
Looks like it needs more work. The result package is too big.

**Comment by snnn** [comment]
Sorry the fix doesn't work

**Pull request closed without merging**